rosdep/python.yaml

```
@@ -8442,6 +8442,10 @@ python3-pyside2.qtwidgets:
     '*': [python3-pyside2]
     '8': null
   ubuntu: [python3-pyside2.qtwidgets]
+python3-pyside6-pip:
+  '*':
+    pip:
+      packages: [PySide6]
```

It seems like PySide6 is coming for next [Ubuntu](https://packages.ubuntu.com/search?keywords=pyside6&searchon=names&suite=all&section=all) and [Debian](https://packages.debian.org/search?keywords=pyside6&searchon=names&suite=all&section=all) distros. Since we rather not have pip packages when the system package are available, I would suggest to scope this to specific distros (feel free to add more):

```suggestion
  debian:
    bullseye:
      pip:
        packages: [PySide6]
    bookworm:
      pip:
        packages: [PySide6]
  ubuntu:
    focal:
      pip:
        packages: [PySide6]
    jammy:
      pip:
        packages: [PySide6]
    noble:
      pip:
        packages: [PySide6]
```

Feel free to add other distros.

---

I have incorporated your suggestion, and also added keys for RHEL 8, RHEL 9, and OSX.